### PR TITLE
feat: add categories

### LINKED ISSUE #18: Add categories

### PR DESCRIPTION
**Pull request by jsmassa**
Closes #18

### CODE REVIEW

File: server/analysis.ts [hunk 1]
```diff
@@ -212,7 +262,7 @@ export async function updateData(dataDir: string, done: (items: Item[]) => void
     items = dedupItems(items);
     writeJSON(`${dataDir}/latest-canonical.json`, items, true);
 
-    pg.insertData(items);
+    //pg.insertData(items);
```
Comment (TimoKramer) [inline server/analysis.ts:265]: Is this change intended?

**Review by TimoKramer — APPROVED** [review]

File: server/analysis.ts (hunk 1 above)
Comment (TimoKramer) [reply server/analysis.ts:265]: I don't know lot of TS so if you want a proper review maybe @kordano ?

Comment (kordano) [reply server/analysis.ts:265]: ok, then I'll do the review.

Comment (jsmassa) [reply server/analysis.ts]: I have removed the comment now

**Review by kordano — APPROVED** [review]
